ui/admin.py

```diff
@@ -123,10 +142,31 @@ def created_at(self, obj):
         return obj.created_at
 
 
+class MoiraListAdmin(admin.ModelAdmin):
+    """admin page of Moira list"""
+    model = models.MoiraList
+    list_display = ('name',)
+    search_fields = ('name',)
+
+
+class VideoSubtitleAdmin(admin.ModelAdmin):
+    """admin page of Moira list"""
+    model = models.VideoSubtitle
+    list_display = ('filename', 'language', )
+    search_fields = ('filename', 'language', 'bucket_name', 'video__titles', )
+
+
+class VideoThumbnailAdmin(admin.ModelAdmin):
+    """admin page of Moira list"""
+    model = models.VideoThumbnail
+    list_display = ('max_width', 'max_height', )
+    search_fields = ('max_width', 'max_height', 'bucket_name', 'video__titles', )
```

Comment:
`max_width` and `max_height` are pretty useless (almost always the same values), they could probably be removed from both `list_display` and `search_fields`.

Might be helpful to put this in `list_display`:
```python
list_display = ('s3_object_key', 'video_id')
```